db/src/urls.rs

```diff
@@ -69,19 +73,29 @@ async fn download_raw_html(url: &str) -> color_eyre::Result<String> {
     Ok(html)
 }
 
+fn clean_raw_html(raw_html: &str, url: &Url) -> color_eyre::Result<String> {
+    let mut raw_html_cursor = std::io::Cursor::new(raw_html);
+    let article = extractor::extract(&mut raw_html_cursor, url)
+        .map_err(|e| color_eyre::eyre::eyre!(e.to_string()))?;
```

Comment:
Do we need the `map_err` here? I think some of this is what the `?` operator with color_eyre does for us so we might not need it 🤞

Reply:
you are correct. I removed it